toolkit/cli/node-commands/src/lib.rs

```diff
@@ -45,9 +94,19 @@ impl CliConfiguration for AriadneParametersCmd {
 	}
 }
 
+/// Command for retrieving fundamental sidechain parameters.
```

Comment:
Somewhat similar to above: what are fundamental sidechain parameters, and if we have fundamental ones what are the non-fundamental ones? hehe